docs/deploy.md

```diff
@@ -149,9 +180,26 @@ authentication behavior for your deployed app.
 
 If you wish to put your Kubernetes Sematic deployment behind SSL, the recommended way to do
 this is to set up an [ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/)
-that points to the service (named `sematic-service`) deployed by the helm chart,
+that points to the service (named `sematic-server`) deployed by the Helm chart,
 and set up your ingress to use SSL.
 
+The Helm chart can create an ingress for you by setting `ingress.create` to `true`, and by
+specifying the ingress domain information like so in the `ingress.hosts` setting:
+
+```
+  hosts:
+    - host: chart-example.local
+      paths:
+        - path: /
+          pathType: ImplementationSpecific
+```
+
+##### Additional configuration options
+
+There are several other configuration options that can be used to customize the
+Sematic Helm installation to your specific needs.  While we've chosen reasonable
+default values for these, you can explore them further by <<TODO: link to Helm values README.md>>.
```

Comment:
Please fill this in.

Reply:
https://github.com/sematic-ai/helm-charts/pull/5